### ISSUE TITLE
Session not being created in 5.1.2

### ISSUE THREAD

**Issue by chrisgilmerproj** [99designs/aws-vault#501]
I just upgraded from 5.0.1 to 5.1.2. The biggest change I noticed is that I now have to use my MFA token on every invocation of the tool. When looking in my Keychain Access in MacOS I no longer see a session being created like I had before, no matter how many times I run the command.  Here is my command with debug:

```sh
$ aws-vault --debug exec PROFILE --duration=1h -- aws s3 ls
2020/01/17 19:43:37 [keyring] Considering backends: [keychain pass file]
2020/01/17 19:43:37 Loading config file /Users/cgilmer/.aws/config
2020/01/17 19:43:37 Parsing config file /Users/cgilmer/.aws/config
2020/01/17 19:43:37 Using region "us-west-2" from AWS_DEFAULT_REGION
2020/01/17 19:43:37 [keyring] Querying keychain for service="aws-vault", keychain="login.keychain"
2020/01/17 19:43:37 [keyring] Found 4 results
2020/01/17 19:43:37 profile PROFILE: using stored credentials
2020/01/17 19:43:37 profile PROFILE: using AssumeRole (using MFA)
Enter token for arn:aws:iam::XXXXX:mfa/cgilmer: XXXXX
2020/01/17 19:43:41 Looking up keyring for 'PROFILE'
2020/01/17 19:43:41 [keyring] Querying keychain for service="aws-vault", account="PROFILE", keychain="login.keychain"
2020/01/17 19:43:42 [keyring] Found item "aws-vault (PROFILE)"
2020/01/17 19:43:42 Generated credentials ****************DEOK using AssumeRole, expires in 1h0m12.509135s
2020/01/17 19:43:42 Setting subprocess env: AWS_DEFAULT_REGION=us-west-2, AWS_REGION=us-west-2
2020/01/17 19:43:42 Setting subprocess env: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
2020/01/17 19:43:42 Setting subprocess env: AWS_SESSION_TOKEN, AWS_SECURITY_TOKEN
2020/01/17 19:43:42 Setting subprocess env: AWS_SESSION_EXPIRATION
```

Can I provide other details?

**Comment by chrisgilmerproj** [comment]
This may be related to https://github.com/99designs/aws-vault/pull/498 but I haven't been able to figure it out on my own yet.

**Comment by mtibben** [comment]
Fixed in 13d41f1dbcf4c97a0a697aaf6a28bfb12f415a29

**Comment by chrisgilmerproj** [comment]
Thank you for the quick turn around! 🎉
